src/components/SelectableMainTable.tsx

```diff
@@ -57,12 +66,15 @@ const SelectableMainTable: FC<Props> = ({
   };
 
   const selectPage = () => {
-    setSelectedNames(rows.map((row) => row.name ?? ""));
+    let selectableRows = rows;
+    selectableRows = rows.filter((row) => !!row.name);
+    setSelectedNames(selectableRows.map((row) => row.name ?? ""));
```

Comment:
```suggestion
    const allNames = rows
      .filter((row) => !!row.name)
      .map((row) => row.name ?? "");
    setSelectedNames(allNames);
```
Maybe a bit more straight forward?